guides/common/modules/ref_supported-operating-systems.adoc

```diff
@@ -26,7 +26,6 @@ ifdef::foreman-deb[]
 endif::[]
 |====
 
-{Team} advises against using an existing system because the {Project} installer will affect the configuration of several components.
```

Comment:
This triggered a change in the Quickstart guide, meaning that the advice to use a fresh system is no longer included in Quickstart. I'm not sure if that's a problem.

Reply:
IMO not a problem.